requirements.txt

```
@@ -12,4 +12,8 @@ reproject
 dust_extinction
 gala
 synphot
+<<<<<<< HEAD
 pyvo==1.2.0
+=======
+pvextractor
+>>>>>>> a911f0e... add pvextractor to reqs
```

```suggestion
pyvo==1.2.0
pvextractor
```

@keflavich Could you resolve this merge conflict here in requirements.txt? I think that should do it.